test/performance/array/distCreate.chpl

```diff
@@ -15,6 +15,14 @@ config const memFraction = 4;
 
 config const correctness = false;
 config const commCount = false;
+config const verboseComm = false;
+config const verboseMem = false;
+
```

Comment:
Not that it matters for a test, but since the above modes are mutually exclusive I wonder if this would be cleaner with a mode enum. e.g. 

```chpl
enum mode {correctness, commCounts, verboseComm, verboseMem}
```

Reply:
You're right. We are using enums in this test for configuration anyways. I'll make the change to keep the implementation consistent and clean

Reply:
(and maybe add `performance` to make it more complete)